doc/api/fs.md

```diff
@@ -3072,6 +3072,32 @@ extended part is filled with null bytes (`'\0'`):
 
 If `len` is negative then `0` will be used.
 
+The `fs.read()` method reads data from the file specified by the file
```

Comment:
Pretty sure this must be moved to `fs.read()` 😄

Reply:
I deleted this place

Reply:
No, the *whole* change must be moved to `fs.read()`

Reply:
I'm so sorry, I thought we were going to keep this place separate